decayAmplitude/isobarDecayTopology.cc

```diff
@@ -543,126 +552,102 @@ isobarDecayTopology::getIsospinSymmetrization()
 
 	// Permutating all the particles and checking if the permutation makes sense
 	//
-	// First, loop over all the groups.
-	for(unsigned int i = 0; i < groups.size(); ++i) {
-		vector<unsigned int> group = groups.at(i);
-
-		// Again debug output
-		if(_debug) {
-			printDebug << "Group permutations " << i << endl;
+	const std::vector< std::vector< std::vector<unsigned int> > > permutations = isobarDecayTopologyHelper::getAllGroupPermutations( groups );
+	for( unsigned int i_permutation = 0; i_permutation < permutations.size(); ++i_permutation ){
+		const std::vector< std::vector<unsigned int> >& permutation = permutations[i_permutation];
```

Comment:
Whitespace coding style:
`for(unsigned int i_permutation = 0; i_permutation < permutations.size(); ++i_permutation) {
		const std::vector<std::vector<unsigned int> >& permutation = permutations[i_permutation];`